### PR TITLE
Filter zero bytes from CRaSH input stream.

### PR DESCRIPTION
**Pull request by chrisr3**
When running Corda via DemoBench, pressing the CTRL, SHIFT, ALT etc keys send an extra zero byte to the Corda CRaSH shell's Standard Input stream. This causes CRaSH mistakenly to map (say) `SHIFT-g` to `BEGINNING_OF_LINE`, or `SHIFT-i` to `BEGINNING_OF_HISTORY`.

The problem is almost certainly a JDK key-handling bug when embedding a Swing component (i.e. JediTerm) in JavaFX, because this does not happen when embedding JediTerm in Swing.

Solve the problem by having DemoBench filter any zero bytes out of Standard Input before they are sent to Corda.

### CODE REVIEW

File: config/dev/log4j2.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
             <AppenderRef ref="RollingFile-Appender" />
         </Logger>
         <Logger name="org.apache.activemq.artemis.core.server" level="error" additivity="false">
-            <AppenderRef ref="RollingFile-Appender"/> 
+            <AppenderRef ref="RollingFile-Appender"/>
         </Logger>
```
Comment (chrisr3) [inline config/dev/log4j2.xml]: Someone added a trailing garbage character here.

File: tools/demobench/src/test/java/net/corda/demobench/getinput/GetInput.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package net.corda.demobench.getinput;
```
Comment (mikehearn) [inline tools/demobench/src/test/java/net/corda/demobench/getinput/GetInput.java]: This file isn't a unit test, right, so perhaps we should just remove it?

File: tools/demobench/src/test/kotlin/net/corda/demobench/pty/JediFXTest.kt [hunk 3]
```diff
@@ -0,0 +1,78 @@
+package net.corda.demobench.pty
```
Comment (mikehearn) [inline tools/demobench/src/test/kotlin/net/corda/demobench/pty/JediFXTest.kt]: I think this won't do what you expect - it'll still be a part of every unit test run, just that failures will be ignored? I get that this was useful to write but I'm not sure having it a part of the repo really helps.

Comment (chrisr3) [inline tools/demobench/src/test/kotlin/net/corda/demobench/pty/JediFXTest.kt]: It won't be run as part of the build. The build would hang if it were.

File: tools/demobench/src/test/java/net/corda/demobench/getinput/GetInput.java (hunk 2 above)
Comment (chrisr3) [inline tools/demobench/src/test/java/net/corda/demobench/getinput/GetInput.java]: That's the source for the `getinput.jar` application, as used by `JediSwingTest` and `JediFXTest`.

**Pull request merged**